std/datetime/mod.ts

```diff
@@ -110,6 +110,30 @@ export function dayOfYear(date: Date): number {
   return Math.floor(diff / DAY);
 }
 
+/**
+ * Get number of the week in the year (ISO-8601)
+ * @return Number of the week in year
+ */
+export function weekOfYear(date: Date): number {
+  const workingDate = new Date(
+    Date.UTC(date.getFullYear(), date.getMonth(), date.getDate())
+  );
```

Comment:
@jacobgc @ry Shouldn't this be called `ISOWeekOfYear`? [Week numbers are locale-specific](https://en.wikipedia.org/wiki/ISO_week_date#Differences_to_other_calendars), and here, this function converts the date to a UTC date, discarding all time information.